<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>transparencySliders</class>
 <widget class="QWidget" name="transparencySliders">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>677</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="Slices">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>341</width>
     <height>671</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>TF</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>9</y>
       <width>403</width>
       <height>346</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Transfer offset</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="TransferOffset">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Transfer Scale</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="TransprencySlider">
        <property name="value">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Sampling Distance</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="SamplingRateSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QCheckBox" name="Boundingbox">
          <property name="text">
           <string>Bounding box</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="AmbientOcclusion">
          <property name="text">
           <string>Ambient Occlusion</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="Preintegration">
          <property name="text">
           <string>PreIntegration</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>401</width>
       <height>251</height>
      </rect>
     </property>
     <widget class="QFrame" name="TransferFunctionWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>401</width>
        <height>251</height>
       </rect>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_4">
       <property name="geometry">
        <rect>
         <x>-1</x>
         <y>-1</y>
         <width>331</width>
         <height>251</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="INteractiveWidgetLayout">
        <item>
         <widget class="QDockWidget" name="dockWidget">
          <widget class="QWidget" name="dockWidgetContents"/>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>TF files</string>
    </attribute>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
    <widget class="QPushButton" name="TFFiles">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>80</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="TransferFunctionPAth">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Slices</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>9</y>
       <width>321</width>
       <height>361</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Slice X Left</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="SliceXleft">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Slice X Right</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="SliceXright">
        <property name="value">
         <number>99</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Slice Y Left </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="SliceYleft">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>Slice Y Right</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="SliceYright">
        <property name="value">
         <number>99</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Slice Z Left</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="SliceZleft">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>Slice Z Right</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="SliceZright">
        <property name="value">
         <number>99</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Phongs </string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>281</width>
       <height>307</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Ambient Light</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="AmbientCheckbox">
          <property name="text">
           <string>Ambient Light</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QSlider" name="AmbientLightSLider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Specular Light </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="SpeculaCheckbox">
          <property name="text">
           <string>Specular Light</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QSlider" name="SpecularLightSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Diffuse light</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="DiffuseCheckbox">
          <property name="text">
           <string>Diffuse light</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QSlider" name="DiffuseSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Shininess (Specular Coefficient)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="shineSlider">
        <property name="value">
         <number>80</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Ray step size (Ambient Occlusion)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="SlideAmbienOcclusionSlider">
        <property name="sliderPosition">
         <number>3</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_7">
    <attribute name="title">
     <string>Dataset</string>
    </attribute>
    <widget class="QTextEdit" name="Dataset_Path">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="OKButtonDataset">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>220</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>220</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>151</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter XYZ dimensions: </string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="x">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="y">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>100</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="z">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>100</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Snap</string>
    </attribute>
    <widget class="QPushButton" name="Snapshot">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>131</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Take Snapshot</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>TransprencySlider</tabstop>
  <tabstop>SamplingRateSlider</tabstop>
  <tabstop>Slices</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>TFFiles</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>TransferFunctionPAth</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
